src/vm/CMakeLists.txt

```diff
@@ -487,6 +487,7 @@ endif()
 if(CLR_CMAKE_PLATFORM_UNIX)
     list(APPEND VM_SOURCES_WKS_ARCH
         ${ARCH_SOURCES_DIR}/unixstubs.cpp
+        unixlowmem.cpp
```

Comment:
This should be added to VM_SOURCES_WKS instead. The VM_SOURCES_WKS_ARCH is meant for architecture specific files.

Reply:
It seems you have forgotten to remove the file from the list here.